code/modules/power/singularity/particle_accelerator/particle.dm

```diff
@@ -27,32 +29,39 @@
 	source = null
 	movetotarget = 1
 
+
 /obj/effect/accelerated_particle/weak
 	movement_range = 8
-	energy = 5
+	energy = PARTICLE_ENERGY / 2
```

Comment:
This is not better, you're still repeating the value, just in a different way.
This should be WEAK_PARTICLE_ENERGY (which you _could_ define as PARTICLE_ENERGY/2) but the calculation itself should still only occur once.